src/control/cmd/daos/pool.go

```diff
@@ -79,8 +79,8 @@ func (cmd *poolBaseCmd) connectPool() error {
 		defer freeString(cLabel)
 
 		cmd.log.Debugf("connecting to pool: %s", cmd.PoolID().Label)
-		rc = C.daos_pool_connect_by_label(cLabel, cSysName,
-			C.DAOS_PC_RW, &cmd.cPoolHandle, &poolInfo, nil)
+		rc = C.daos_pool_connect(cLabel, cSysName, C.DAOS_PC_RW,
```

Comment:
Out of curiosity I tried this out and it does seem to go through the new macro and land at the new daos_pool_connect2() function. But I wonder if especially in a cross-language call if we should instead directly call the new function?

Reply:
hm, if it ended up into daos_pool_connect2(), then it means that it went through the macro, no?

Reply:
sorry, i misread your comment.

Reply:
It works with python and go .... at least